Bojzen-I-Mitten/Stort-spelprojekt
# System/objectmanagement

Changelog:
+ Multi-threaded asset loading 
+ Changed Editor BootUp
+ Removed component locks
+ Checks for object initiation improved (object is deleted if failed initiation)
+ Fixed double register on copy/paste

HowToTest:
+ Try to break editor, most changes (should not) affect runtime
+ Verify that assets are loaded correctly in external build
